plugins/DuskVerb/tools/perceptual_diff.py

```
@@ -0,0 +1,1077 @@
+# ─── Main ─────────────────────────────────────────────────────────────────
+
+def main():
+    args = sys.argv[1:]
+    if len(args) >= 4:
+        lex_ir_path, dv_ir_path, lex_nb_path, dv_nb_path = args[:4]
+    elif len(args) == 2:
+        lex_ir_path, dv_ir_path = args
+        lex_nb_path = lex_ir_path.replace('_impulse', '_noiseburst')
+        dv_nb_path = dv_ir_path.replace('_impulse', '_noiseburst')
+    else:
+        lex_ir_path = '/tmp/lex_compare/lex_v3_impulse.wav'
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/lex_compare/lex_v3_impulse.wav"

(S108)